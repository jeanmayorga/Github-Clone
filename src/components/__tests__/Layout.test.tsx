import { render } from '@testing-library/react'
import { Layout } from '..'

describe('Render layout', () => {
  it('should render layout', () => {
    const { container } = render(
      <Layout>
        <div>Im a children</div>
      </Layout>
    )
    expect(container.firstChild).toMatchSnapshot()
  })
})
