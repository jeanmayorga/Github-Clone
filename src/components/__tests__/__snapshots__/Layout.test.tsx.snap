// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render layout should render layout 1`] = `
<div
  class="bg-white dark:bg-github-dark-bg text-sm dark:text-github-dark-text"
>
  <div
    class="bg-github-header dark:bg-github-dark-header h-[62px]"
  />
  <div>
    Im a children
  </div>
</div>
`;
